libs/api/domains/cms/src/lib/services.ts

```diff
@@ -352,3 +352,12 @@ export const getLifeEventPage = async (
 
   return result.items.map(mapLifeEventPage)[0] ?? null
 }
+
+export const getLifeEvents = async (lang: string): Promise<LifeEventPage[]> => {
+  const result = await getLocalizedEntries<types.ILifeEventPageFields>(lang, {
+    ['content_type']: 'lifeEventPage',
+    order: '-sys.createdAt',
+  }).catch(errorHandler('getLifeEventPage'))
```

Comment:
should be getLifeEvents so the error handler reports the correct function is something goes wrong